### PR TITLE
Removing internals visible to SharedUx in Axe.Windows.Actions.

### PR DESCRIPTION
**Pull request by RobGallo**
It seems safe to give external code access to get, but not set, an A11yElement. Perhaps the A11yElement has code that should not be exposed either, but that is beyond the scope of this change.

#### Describe the change
(Please provide an overview of the change in this PR)

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights.Actions/Actions/DataManager.cs [hunk 1]
```diff
@@ -131,6 +132,35 @@ internal A11yElement GetA11yElement(Guid ecId, int eId)
...
         }
 
+        /// <summary>
+        /// Gets the element on which the screenshot was taken.
+        /// </summary>
+        /// <param name="ecId"></param>
+        /// <returns></returns>
+        public A11yElement GetScreenshotElement(Guid ecId)
+        {
+            var elementContext = GetElementContext(ecId);
```
Comment (DaveTryon) [inline src/AccessibilityInsights.Actions/Actions/DataManager.cs]: >var [](start = 12, length = 3)

Consider null propagation?

var dc = GetElementContext(ecId)?.DataContext;
if (dc == null) return null
return GetA11yElement(ecId, dc);

File: src/AccessibilityInsights.Actions/Actions/DataManager.cs [hunk 2]
```diff
@@ -131,6 +132,35 @@ internal A11yElement GetA11yElement(Guid ecId, int eId)
...
+        }
+
+        /// <summary>
+        /// Gets the most recent screenshot
+        /// </summary>
+        /// <param name="ecId"></param>
+        /// <returns></returns>
+        public Bitmap GetScreenshot(Guid ecId)
+        {
+            var elementContext = GetElementContext(ecId);
```
Comment (DaveTryon) [inline src/AccessibilityInsights.Actions/Actions/DataManager.cs]: >var [](start = 12, length = 3)

Consider null propagation?

return GetElementContext(ecId)?.DataContext?.Screenshot;

**Comment by DaveTryon** [comment]
>         public int? FocusedElementUniqueId { get; set; }

Should this setter still be public?

---
Refers to: src/AccessibilityInsights.Actions/Contexts/ElementDataContext.cs:50 in 99ede04. [](commit_id = 99ede04313efe2894bb00bbd135714d3c4f557b3, deletion_comment = False)

**Comment by RobGallo** [comment]
> > ```
> >     public int? FocusedElementUniqueId { get; set; }
> > ```
> 
> Should this setter also be internal?
> 
> Refers to: src/AccessibilityInsights.Actions/Contexts/ElementDataContext.cs:50 in [99ede04](https://github.com/Microsoft/accessibility-insights-windows/commit/99ede04313efe2894bb00bbd135714d3c4f557b3). [](commit_id = [99ede04](https://github.com/Microsoft/accessibility-insights-windows/commit/99ede04313efe2894bb00bbd135714d3c4f557b3), deletion_comment = False)

When I tried making it internal, the code ceased to compile.

**Pull request merged**